package.json

```diff
@@ -11,6 +11,9 @@
     "type": "git",
     "url": "git+https://github.com/keeper335/node_gmp_homework1.git"
   },
+  "dependencies": {
+    "node-csvtojson": "git://github.com/Keyang/node-csvtojson"
```

Comment:
it's better to use packages from npm. if the repository is deleted, you won't be able to use this pacakge

Reply:
This is just a way to comply task requirements to use the mentioned link as is